cumulusci/tasks/push/tests/test_push_tasks.py

```diff
@@ -0,0 +1,481 @@
+        assert task._load_orgs_file(ORG_FILE) == []
+
+    # testing with multiple orgs
+    with open(ORG_FILE, "r") as file:
+        assert task._load_orgs_file(ORG_FILE) == [
+            "00DS0000003TJJ6MAO",
+            "00DS0000003TJJ6MAL",
+        ]
+    os.remove(ORG_FILE)
+
+
+def test_get_subs_raises_err():
+    task = create_task(GetSubscriberList, options={"filename": "in.csv"})
+    with pytest.raises(NotImplementedError):
+        task()
+
+
+def test_filter_subs_raises_err():
+    task = create_task(
+        FilterSubscriberList, options={"file_in": "in.csv", "file_out": "out.txt"}
+    )
+    with pytest.raises(NotImplementedError):
+        task()
+
+
+def test_base_push_task_raises_err():
+    task = create_task(BaseSalesforcePushTask, options={})
+    with pytest.raises(NotImplementedError):
+        task()
+
+
+def test_report_push_status_error():
+    task = create_task(BaseSalesforcePushTask, options={})
+    task.sf = mock.MagicMock()
+    task.push_report = mock.MagicMock()
+    task.sf.query_all.return_value = {"totalSize": 1, "records": []}
+    with pytest.raises(PushApiObjectNotFound):
+        task._report_push_status("0DV1R000000k9dEWAQ")
+
+
+def test_get_push_request_job_results(
+    package_push_job_success, package_push_job_failure, package_push_job_cancel
+):
+    task = create_task(BaseSalesforcePushTask, options={})
+    task.sf = mock.MagicMock()
+    task.push_report = mock.MagicMock()
+    task.push_request = mock.MagicMock()
+    task.push_request.get_push_job_objs.return_value = [
+        package_push_job_success,
+        package_push_job_failure,
+        package_push_job_cancel,
+    ]
+    task._get_push_request_job_results()
+    task.push_request.get_push_job_objs.assert_called_once()
+
+
+def test_schedule_push_org_query_get_org_error():
+    task = create_task(
+        SchedulePushOrgQuery,
+        options={
+            "orgs": ORG,
+            "version": VERSION,
+            "namespace": NAMESPACE,
+            "start_time": None,
+            "batch_size": "200",
+            "min_version": "1.1",
+        },
+    )
+    task.push = mock.MagicMock()
+    task.push_api = mock.MagicMock()
+    task.sf = mock.Mock()
+    task.push_api.return_query_records = mock.MagicMock()
+    task.push_api.return_query_records.return_value = {"totalSize": "1"}
+    task.sf.query_all.return_value = PACKAGE_OBJS
+    with pytest.raises(ValueError):
+        assert task._get_orgs() == [NAME]
+
+
+def test_schedule_push_org_query_get_org():
+    task = create_task(
+        SchedulePushOrgQuery,
+        options={
+            "orgs": ORG,
+            "version": VERSION,
+            "namespace": NAMESPACE,
+            "start_time": None,
+            "batch_size": "200",
+            "subscriber_where": "OrgType = 'Sandbox'",
+        },
+    )
+    task.push = mock.MagicMock()
+    task.push_api = mock.MagicMock()
+    task.sf = mock.Mock()
+    task.sf.query_all.return_value = PACKAGE_OBJS
```

Comment:
Your mock return value doesn't make sense here. The code being tested is doing a query for PackageSubscribers, but PACKAGE_OBJS contains a response with PackagePushRequests.